docs/understanding-your-pipeline/events/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Understanding events"
+sidebar_label: "Events"
+sidebar_position: 1
+description: "An event is a central concept in Snowplow that represents something that occurred in a particular point in time"
+---
+
+An event is something that occurred in a particular point in time. Examples of events include:
+
+- Load a web page
+- Add an item to basket
+- Enter a destination
+- Check a balance
+- Search for an item
+- Share a video
+
+## Kinds of events
+
+At the high level, there are 3 kinds of Snowplow events:
+* [Standard events](#baked-in-events) that are very common and therefore “baked in”, for example, page views
+* [Structured events](#structured-events), which you can supplement with some (limited) custom information
+* [Self-describing events](#self-describing-events), which can include arbitrarily complex data, described by a [schema](/docs/understanding-your-pipeline/schemas/index.md)
+
+You can create all of these by using various [tracking SDKs](/docs/collecting-data/collecting-from-own-applications/index.md).
+
+In the data warehouse, all 3 kinds of events share a number of standard columns, such as timestamps. That said, event-specific data will be stored differently, as explained below. See also [what Snowplow data looks like](/docs/understanding-your-pipeline/canonical-event/index.md).
+
+## Out-of-the-box and custom events
+
+Snowplow supports a large number of events out of the box, for example:
+* Page views and screen views
+* Page pings
+* Link clicks
+* Form fill-ins (for the web)
+* Form submissions
+* Transactions
+
+Some of these are [baked-in events](#baked-in-events), while others are [self-describing events](#self-describing-events) that were predefined by the Snowplow team, e.g. link clicks. Tracking SDKs usually provide dedicated API to create out-of-the-box events (regardless of their kind).
+
+You can also create custom events to match your business logic. For that purpose, you can either define your own [self-describing events](#self-describing-events) (recommended), or use [structured events](#structured-events).
```

Comment:
Maybe "business requirements" or "own requirements" over "business logic"? I feel like "business logic" implies some kind of action/process that event definitions aren't really related to.